src/components/StartupService.js

```diff
@@ -90,6 +90,9 @@ const StartupService = ( ) => {
       };
       // don't remove this logger.info statement: it's used for internal metrics
       logger.info( "pickup" );
+      logger.info(
+        `App version: ${DeviceInfo.getVersion()}, build: ${DeviceInfo.getBuildNumber()}`
```

Comment:
This makes looking at log files a bit easier, so we know what version of the code they are actually running.